Engine/Makefile

```diff
@@ -41,13 +49,18 @@ common.a: $(OBJS_COMMON)
 	@echo $@
 	$(CMD_PREFIX) $(CC) $(CFLAGS) -MD -c -o $@ $<
 
+%.o: %.mm
```

Comment:
do we really want to create dependency information? in my experience this often makes the build almost 2x slower. and the advantage added by this is questionable.

Reply:
First off, this is to support objective-c code.  We already build dependency information and while it's not perfect, it does help when I'm doing development and incremental builds.